nginx/metadata.csv

```diff
@@ -179,5 +179,12 @@ nginx.upstream.peers.sent,gauge,,byte,,The total amount of data sent to this ser
 nginx.upstream.peers.sent_count,count,,byte,,The total amount of data sent to this server (shown as count).,0,nginx,upstr peers sent
 nginx.upstream.peers.unavail,gauge,,,,How many times the server became unavailable for client requests (state "unavail") due to the number of unsuccessful attempts reaching the max_fails threshold.,0,nginx,upstr peers unavail
 nginx.upstream.peers.unavail_count,count,,,,How many times the server became unavailable for client requests (state "unavail") due to the number of unsuccessful attempts reaching the max_fails threshold (shown as count).,0,nginx,upstr peers unavail
-nginx.upstream.peers.weight,gauge,,,,Weight of the server.,0,nginx,upstr peers weight
```

Comment:
why is this one removed?

Reply:
My bad, I had named the metrics nginx.upstream, seems I was too greedy when I fixed.